### PR TITLE
Add ShowMenuToggle and update logo description

### PR DESCRIPTION
**Pull request by almenscorner**
Updated with Support Companion 1.0.5 additions and changes.

### CODE REVIEW

File: Manifests/ManagedPreferencesApplications/com.almenscorner.supportcompanion.plist [hunk 1]
```diff
@@ -278,6 +279,16 @@
...
 		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>Configures whether to show the menu toggle button in the apps side menu.</string>
+			<key>pfm_name</key>
+			<string>ShowMenuToggle</string>
+			<key>pfm_title</key>
+			<string>Show Menu Toggle</string>
+			<key>pfm_type</key>
+			<string>boolean</string>
```
Comment (relgit) [inline Manifests/ManagedPreferencesApplications/com.almenscorner.supportcompanion.plist:290]: The interaction with boolean properties really benefits from them having default values (`pfm_default_value`). Do you mind adding it here?

Take this opportunity to add one to `IntuneMode` too if you can.

**Review by relgit — CHANGES_REQUESTED** [review]
Thanks for the update @almenscorner .

See my comment please.

File: Manifests/ManagedPreferencesApplications/com.almenscorner.supportcompanion.plist (hunk 1 above)
Comment (almenscorner) [reply Manifests/ManagedPreferencesApplications/com.almenscorner.supportcompanion.plist:290]: I think I added what you suggested here :)

**Review by relgit — APPROVED** [review]
Thanks @almenscorner. This looks good.

I'm just going to change the key sorting order and then I can merge.

**Pull request merged**